scripts/presets/travis/replica_sets/replicaset.json

```diff
@@ -0,0 +1,66 @@
+{
+    "id": "REPLICASET",
+    "name": "mongod",
+    "members": [
+        {
+            "procParams": {
+                "dbpath": "/tmp/REPLICASET/3000/",
+                "ipv6": true,
+                "logappend": true,
```

Comment:
Is this option important given that the Travis build boxes are ephemeral?

Reply:
It was useful for Travis in debug mode.